src/xorIt.js

```diff
@@ -10,23 +6,19 @@
  * http://cwestblog.com/2013/08/19/javascript-simple-vernam-cipher/
  */
 
-var xorIt = (key, textOrCipher) => {
+export default function xorIt(key, textOrCipher) {
   if (!key || !textOrCipher) {
     throw 'XorIt: needs key and textOrCipher';
   }
 
-  let cipher = [],
-      keyLength = key.length,
-      fromCharCode = String.fromCharCode,
-      len = textOrCipher.length,
-      i = 0;
+  let cipher = [];
```

Comment:
Nope because this variable is being modified within the for loop. Basically ESlint is dumb whenever variable is changed inside of for... loops or if... statements and other situations. Hence it is just a warning here.